src/com.pentalog.pentastagiu/week3/Bank.java

```diff
@@ -0,0 +1,60 @@
+package com.pentalog.pentastagiu.week3;
+
+public class Bank {
+    Scanner input = new Scanner(System.in);
```

Comment:
nu cred ca ai comis tot ce era nevoie cand ai facut pull request-ul, mie nu imi compileaza proiectul:
- pare sa lipseasca importurile (de ex Scanner)
- nu exista clasa BankAccount in clasele comise de tine
- nu ai un punct de intrare pentru a testa aplicatia (metoda public static void main): codul tau e pus direct in clasa Bank si asta e un motiv in plus pentru care nu compileaza